Description: Remove use of prettify in javadoc as it provided
 in non-preferrred compressed source and adds minimal value.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene-solr/lucene/common-build.xml
===================================================================
--- lucene-solr.orig/lucene/common-build.xml	2012-06-07 12:25:01.778670365 +0100
+++ lucene-solr/lucene/common-build.xml	2012-06-07 12:25:59.190842526 +0100
@@ -985,7 +985,6 @@
     <sequential>
       <antcall target="download-java5-javadoc-packagelist"/>
       <delete file="@{destdir}/stylesheet.css" failonerror="false"/>
-      <copy todir="@{destdir}" file="${prettify.dir}/prettify.js" overwrite="false" />
       <record name="@{destdir}/log_javadoc.txt" action="start" append="no"/>
       <javadoc
           overview="@{overview}"
@@ -1011,34 +1010,12 @@
         description="NOTE: This API is for internal purposes only and might change in incompatible ways in the next release."/>
       	<link offline="true" packagelistLoc="${javadoc.dir}"/>
         <link offline="true" href="${javadoc.link}" packagelistLoc="${javadoc.packagelist.dir}/java5"/>
-        <bottom><![CDATA[
-          <script src='{@docRoot}/prettify.js' type='text/javascript'></script>
-          <script type='text/javascript'>
-            (function(){
-              var oldonload = window.onload;
-              if (typeof oldonload != 'function') {
-                window.onload = prettyPrint;
-              } else {
-                window.onload = function() {
-                  oldonload();
-                  prettyPrint();
-                }
-              }
-            })();
-          </script>
-        ]]></bottom>
-      	
       	<sources />
       	      	
         <classpath refid="javadoc.classpath"/>
       </javadoc>
       <record name="@{destdir}/log_javadoc.txt" action="stop"/>
       
-      <!-- append prettify.css -->
-      <concat destfile="@{destdir}/stylesheet.css" append="true">
-        <filelist dir="${prettify.dir}" files="prettify.css"/>
-      </concat>
-
       <delete>
         <fileset file="@{destdir}/log_javadoc.txt">
           <or>
